resources/js/Components/HamburgerMenu.vue

```diff
@@ -21,6 +21,14 @@ const props = defineProps<{
         editions: number[];
     };
 }>();
+
+watch(open, () => {
+    if (open.value) {
+        document.body.classList.add("overflow-hidden");
+    } else {
+        document.body.classList.remove("overflow-hidden");
+    }
```

Comment:
```suggestion
    document.body.classList.toggle("overflow-hidden");
```

Isn't this a cleaner alternative?